sdk/js/src/Client.ts

```diff
@@ -46,7 +46,20 @@ export class DVCClient implements Client {
     private windowMessageHandler?: (event: MessageEvent) => void
 
     constructor(environmentKey: string, user: DVCUser, options: DVCOptions = {}) {
+        this.logger = options.logger || dvcDefaultLogger({ level: options.logLevel })
+        this.store = new Store(typeof window !== 'undefined' ? window.localStorage : stubbedLocalStorage, this.logger)
+        
+        const storedAnonymousId = this.store.store?.getItem(StoreKey.AnonUser)
```

Comment:
it would be better to use `this.store.load` function instead

Reply:
ummm...but it returns a Promise, do we really need a Promise for this case?

Reply:
I can clear all the promises in store in other pr